app/utils/rw_api.py

```diff
@@ -84,6 +84,37 @@ async def who_am_i(token) -> Response:
     return response
 
 
+async def get_rw_user(user_id: str) -> User:
+    """Call RW API to get a user from RW API."""
+
+    headers = {"Authorization": f"Bearer {SERVICE_ACCOUNT_TOKEN}"}
+    url = f"{RW_API_URL}/auth/user/{user_id}"
+
+    try:
+        async with AsyncClient() as client:
+            response: HTTPXResponse = await client.get(
+                url, headers=headers, timeout=10.0
+            )
+    except ReadTimeout:
+        raise HTTPException(
+            status_code=500,
+            detail="Call to user service timed-out. Please try again.",
+        )
+
+    if response.status_code == 404:
+        raise HTTPException(status_code=401, detail=f"User ID invalid: {user_id}")
+
+    if response.status_code != 200 and response.status_code != 401:
```

Comment:
I'm not sure why you are explicitly exempting status_code 401 here.  It seems like that is a real response from /auth/user/{user_id}: https://resource-watch.github.io/doc-api/reference.html#get-the-current-user.  We might get that error if our service account has expired?  In any case, seems like you should raise an exception for any status code other than 200, since response won't be correct otherwise.